### ISSUE TITLE
Possibility to manualy specify the IP address

### ISSUE THREAD

**Issue by sliwma** [sergeymaysak/smartercoffee#5]
It would also be a neat feature, as currently HA has to be in the same network as the coffee maker, and even though it sometimes has problems to connect.

**Comment by sergeymaysak** [comment]
Having the ability to specify IP addresses will not fix connectivity issues, trust me ).  Also, unless you have not specified in your router a reservation for IP address my mac address of coffeemaker (which is too much imo), IP address of coffeemaker may be changed eventually by your router... Current logic handles it for you automatically... So sorry, but IP address will not be exposed as a setting... Keep in mind, that smartercoffee hardware is a bit crappy ) and just losts connection twice a week for me ....

**Comment by sliwma** [comment]
So what would you suggest?
My HA is in VLAN A, Smarter is in VLAN B - I added a second network to HA, so HA in both VLAN A & B, and even with that Smarter got unavailable after few days (toggling Smarter, rebooting HA didn't help), when it is available using iBrew addon, and via the app - and my smartphone is in VLAN C.

**Comment by sergeymaysak** [comment]
I do restart the coffeemaker and do reload of my integration (instead of restart HA) when the coffeemaker hangs... it helps for several days... I've learned that the coffeemaker closes the connection after some time and refuses any further connections - I suspect it leaks sockets/other resources in its firmware... This can be fixed only by the reboot of the coffeemaker. If it works for a few days then your VLAN A & B is fine. reboot/reload should help. If iBrew works in the same environment, I suppose my integration should work as well... anyway - I need some time to review iBrew code on the difference with mine...

**Comment by sergeymaysak** [comment]
@sliwma if time allows - could you please install [latest build](https://github.com/sergeymaysak/smartercoffee/releases/tag/1.4.6) and let me know if it helps

**Comment by sliwma** [comment]
Hey @sergeymaysak, I disabled VLAN A, reset the Coffee machine, we'll see if it will stick after enabling VLAN A. Also, I've noticed that there aren't any entities under "SmarterCoffee Maker" device. I'll let you know if the integration works in a few days.

**Comment by sliwma** [comment]
Yeah, so it took not even 24hrs (19:46 - 19:18) for the Smarter to get unavailable.

**Comment by sergeymaysak** [comment]
Hmmm, my device runs several days already w/o issues. I even made two test HA servers to stress it. Now sure, where to go from this point...
